### ISSUE TITLE
STPPaymentCardTextField postal code entry issue in Russian locale

### ISSUE THREAD

**Issue by lauri-koli-vincit** [stripe/stripe-ios#1600]
## Summary
The STPPaymentCardTextField postal code entry behaves erroneously when the app is set to Russian language. After entering the CVC focus moves to the postal postal code field but the field is out of bounds of the STPPaymentCardTextField. The issue does not appear when using short placeholder for CVC field instead of the default.

[Here's a link](https://drive.google.com/file/d/1Rm19iTxd5qBQNgRV_9B8p5kuCPz95n8z/view?usp=sharing) to a video which shows the incorrect behaviour.

## Code to reproduce

A STPPaymentCardTextField with default configuration except having postal code entry enabled.

```
let field = STPPaymentCardTextField()
field.postalCodeEntryEnabled = true
...
```
## iOS version

Tested iOS 13.5.x.

## Installation method

CocoaPods

## SDK version

Tested on SDK versions 19.2.0 and 19.3.0.

**Comment by yuki-stripe** [comment]
Hi @lauri-koli-vincit, 

Thank you for reporting this bug! We're tracking this internally.

**Comment by lauri-koli-vincit** [comment]
Hi @yuki-stripe,

Any update on this?

**Comment by davidme-stripe** [comment]
This was fixed in [21.2.1](https://github.com/stripe/stripe-ios/releases/tag/21.2.1).

**Comment by davidme-stripe** [comment]
Update: Sorry, but this didn't actually land in 21.2.1! We'll land it in the next update.

**Comment by davidme-stripe** [comment]
This was fixed (for real!) in [21.3.0](https://github.com/stripe-ios/stripe-ios/blob/private/CHANGELOG.md#2130-2021-02-18).
